### ISSUE TITLE
`aria-expanded` property seems to be missing, but appears in devtools

### ISSUE THREAD

**Issue by kaiyoma** [component-driven/cypress-axe#153]
I just started using cypress-axe, and the very first error I'm getting is this:

```
Required ARIA attributes must be provided
=========================================

<input type="search" autocomplete="off" class="ant-select-selection-search-input" role="combobox" aria-haspopup="listbox" aria-owns="rc_select_1_list" aria-autocomplete="list" aria-controls="rc_select_1_list" aria-activedescendant="rc_select_1_list_0" readonly="" unselectable="on" value="" id="rc_select_1" style="opacity: 0;">

Fix any of the following:
  Required ARIA attribute not present: aria-expanded
```

However, when I use devtools to inspect the element in question (a select from Ant Design), I can see that `aria-expanded` is actually there:

![image](https://user-images.githubusercontent.com/10719780/218895793-a8d81ae8-6f3b-4a66-8114-7d7566d0b4d2.png)

This problem is reproducible on all the Ant select components on the page. I'm running `cy.checkA11y` well after the page has fully rendered, so I don't think it's an issue of "the property hasn't shown up yet". The property appears to be there, but cypress-axe is not seeing it for some reason.

**Comment by kaiyoma** [comment]
I'll also add that when I run the WAVE extension in Chrome on the same page, it doesn't report any problems with `aria-expanded`.
